cmd/cloud-controller-manager/app/apis/config/v1alpha1/doc.go

```diff
@@ -21,8 +21,7 @@ limitations under the License.
 
 // +k8s:deepcopy-gen=package
 // +k8s:conversion-gen=k8s.io/kubernetes/cmd/cloud-controller-manager/app/apis/config
-// +k8s:conversion-gen=k8s.io/apimachinery/pkg/apis/config/v1alpha1
-// +k8s:conversion-gen=k8s.io/apiserver/pkg/apis/config/v1alpha1
+// +k8s:conversion-gen=k8s.io/component-base/config/v1alpha1
```

Comment:
is nothing left in the old packages such that we can drop them here?

Reply:
Everything was moved out of them.

Reply:
yep, nothing's left there